qlty-analysis/src/lang.rs

```diff
@@ -3,6 +3,7 @@ use core::fmt;
 use std::sync::Arc;
 use tree_sitter::{Node, Parser, Query};
 
+mod c_sharp;
```

Comment:
I would probably call this `csharp`, because we are going to have `cpp` represent C++ and it would be a bit odd to have that as `c_p_p`